### DISCUSSION TITLE
Prisma v5 supported?

### DISCUSSION THREAD

**Discussion by jbeckton** [amplication/amplication#8116] [Q&A] [answered]
Just cloned the Amplication repo and trying to get it running locally. I bump the Prisma version to 5 since there are a lot of performance enhancements in v5. Now I see lots of ts errors.

**Comment by jbeckton** [comment]
Disregard... the codebase is broken, missing env vars for github. No further interest.

**Reply by overbit** [reply]
@jbeckton only mandatory environment variables (no secrets) are part of the tracked .env file on the packages.
If you'd like to configure GitHub for Auth and sync you can follow the guides on amplication docs site i.e. https://docs.amplication.com/running-amplication-platform/configure-github-auth/

But as mentioned before these are not mandatory as you can use a local account to sign up and login when running locally.

@jbeckton has you mentioned that the codebase is broken, could you please add more details so we can help you more?

**Comment by overbit** [comment, marked as answer]
Currently code generated by the latest version of amplication code generator is created with prisma v5

As the amplication's top priority is its users code, the code generator was migrated first to v5.

The current amplication platform is still on prisma v4 but we are prioritising the migration to v5 soon
I've opened a new issue to track its progress https://github.com/amplication/amplication/issues/8117

**Reply by overbit** [reply]
we merged this in `next` so, it should be live soon
